packages/sanity/src/_internal/manifest/__tests__/Icon.test.tsx

```diff
@@ -0,0 +1,50 @@
+import {studioTheme, ThemeProvider} from '@sanity/ui'
+import {render as renderRTL, screen} from '@testing-library/react'
+import {describe, expect, it} from 'vitest'
+
+import {SchemaIcon, type SchemaIconProps} from '../Icon'
+
+const render = (props?: Partial<SchemaIconProps>) =>
+  renderRTL(<SchemaIcon title="Studio" {...props} />, {
+    wrapper: ({children}) => <ThemeProvider theme={studioTheme}>{children}</ThemeProvider>,
+  })
+
+describe('SchemIcon', () => {
```

Comment:
I didnt realize the code was copypasta, so feel free to ignore.
The things I personally would like to see testing is the colors and title stuff, not the jsx. Maybe what I really wanted was the hard bits to be function extracted and tested in isolation. 
But if you keep the tests, check for the expected fill colors and stuff?

Again, no need to humor me.